cmake/configs/posix_sitl_ekf2.cmake

```diff
@@ -1,99 +1,14 @@
-include(posix/px4_impl_posix)
+include(cmake/configs/posix_sitl_default.cmake)
```

Comment:
This should be a verbatim copy of the default config, except for the startup script, which starts EKF2 instead of LPE.